plugin/impl.go

```diff
@@ -270,12 +135,14 @@ func parseParams(paramList []string) (map[string]string, error) {
 		parts := strings.SplitN(p, "=", 2)
 		if len(parts) == 2 {
 			params[parts[0]] = parts[1]
-		} else if _, err := os.Stat(parts[0]); os.IsNotExist(err) {
-			return nil, fmt.Errorf(
-				"invalid param '%s'; must be KEY=VALUE or file path",
-				parts[0],
-			)
 		} else {
+			if _, err := os.Stat(parts[0]); err != nil {
+				return nil, fmt.Errorf(
+					"invalid param '%s'; must be KEY=VALUE or file path",
+					parts[0],
+				)
+			}
+
```

Comment:
I don't see any benefit for moving the `else if` into this block.

Reply:
I think it's far more understandable when reading, since it's a pain to get your head around in which exact cases the `else` block would originally fire. You also get the idiomatic Go error handling with early return on an error, and no corresponding `else`.